### ISSUE TITLE
Triple money stacks / Cannot give/transfer cash

### ISSUE THREAD

**Issue by neoreturns** [dutchplayers/FiveM-Resources#22]
Hello! First of all thanks for your work, its fantastic - im working on a couple of issues at the moment, may even have to code a lazy workaround e.g /givecash for transferring money between players. 

Could you point me in the right direction for why these issues may happen? 

Then I may be able to use my previous experience to help fix them.

**Comment by RuisSoftware** [comment]
Hi! Thank you for using this resource.

I think you are experiencing the same bug we've had couple weeks ago. I am not sure how i fixed it those days.

I'll try to reproduce it and come back with info.

If you need other support you can visit our Dutch Discord server here: https://www.dutch-players.nl/go/discord/

**Comment by RuisSoftware** [comment]
I've found a solution for now.

Config.IncludeCash = true -- DONT TOUCH!
Config.IncludeWeapons = true -- TRUE or FALSE
Config.IncludeAccounts = true -- TRUE or FALSE
Config.ExcludeAccountsList = {"bank", "money"} --  DONT TOUCH!

This is how your config should look like. You can now give money, but not duplicated

**Comment by prajeetware** [comment]
Hey dutchplayers, thanks for your reply! I will test the updated merge. Again thanks for your work!

**Comment by neoreturns** [comment]
This is the config setup I'm currently using, is there any new updates elsewhere to reflect this change?

**Comment by neoreturns** [comment]
Since the latest merge, moving money into trunks, gloveboxes and even when force opening other players inventory works. But using 'GIVE' function does not transfer the money after successfully finding nearby players.

**Comment by ClicheUsername** [comment]
@prajeetware I am working on a fix for the "Give" function issue. Once I can get another person to be able to test it with I'll make sure it works and then put in a pull request.

**Comment by neoreturns** [comment]
@ClicheUsername Happy to assist with testing if you need it!

**Comment by ClicheUsername** [comment]
@prajeetware Sure, add me on Discord: ClickyUsername#1337

**Comment by ClicheUsername** [comment]
@prajeetware Sure, add me on Discord: ClickyUsername#1337

**Comment by Rado236** [comment]
any update?

**Comment by neoreturns** [comment]
Yes, money transfers is fixed, but this causes a new issue with theft scripts, will fix today.
Triple money stacks can be easily fixed by removing the do for all keypairs in accounts, when there is only one account :) Thanks!

**Comment by ghost** [comment]
Any updates?

**Comment by ClicheUsername** [comment]
I put in a PR that will close this issue. Once #49 is merged make sure you download the latest copy.
